hub/core/storage/tests/test_benchmark_storage_provider.py

```diff
@@ -1,23 +1,22 @@
 import pytest
 
-from hub.constants import MB
+from hub.constants import GB
 from hub.tests.common_benchmark import (
     parametrize_benchmark_chunk_sizes,
     BENCHMARK_CHUNK_SIZES,
 )
 from hub.core.tests.common import parametrize_all_caches, parametrize_all_storages
 from hub.core.storage.tests.test_storage_provider import KEY  # type: ignore
 
-SIMULATED_DATA_SIZES = [128 * MB]
+
+SIMULATED_DATA_SIZES = [1 * GB]
```

Comment:
please do not use 1 GB of data here (128 * MB is sufficient)

Reply:
this is probably why the tests are failing (they are timing out, presumably due to this taking a long time)